src/components/Stake/GMXAprTooltip.tsx

```diff
@@ -52,8 +52,11 @@ export default function GMXAprTooltip({ processedData, nativeTokenSymbol }: Prop
             showDollar={false}
             value={`${formatKeyAmount(processedData, "gmxAprForNativeTokenWithBoost", 2, 2, true)}%`}
           />
-          <br />
-          {renderEscrowedGMXApr(processedData)}
+          {renderEscrowedGMXApr(processedData) && (
```

Comment:
double rendering looks weird, let's store`renderEscrowedGMXApr(processedData)` in the variable

Reply:
Updated!